deployment/2_deploy_contracts.js

```diff
@@ -59,7 +59,7 @@ async function main() {
 
   // Deploy Module Registry
   const ModuleRegistryWrapper = await ModuleRegistry.new();
-  const AuthoriserWrapper = await Authoriser.new();
+  const AuthoriserWrapper = await Authoriser.new(newConfig.settings.timelockPeriod);
```

Comment:
Since this contract is deployed for the first time and is part of the 2.5 release I feel this line should be part of the `7_upgrade_2.5.js` file .